src/components/App.css

```diff
@@ -0,0 +1,11 @@
+body {
+    background-color: #e8eaf6;
+}
+
+h1 {
+    margin-top: 2vh !important;
```

Comment:
Do you actually need here `!important` css property? You could consider using a more specific selector instead of `h1`, for example `h1.my-title`.